### DISCUSSION TITLE
Permissions to claim in certain worlds (override config)

### DISCUSSION THREAD

**Discussion by WillFP** [GriefPrevention/GriefPrevention#683] [Q&A] [answered]
Title pretty much says it, be able to claim in worlds where it isn't usually allowed
Permissions could be griefprevention.world.world, griefprevention.world.nether, etc etc
Would this be possible?

**Comment by Jikoo** [comment, marked as answer]
You can already do this by using a permissions plugin that allows per-world permissions. Enable claims in the world and negate `griefprevention.claims` from groups that should not be able to claim.
